changelog.txt

```diff
@@ -8,6 +8,7 @@
 * Tweak - Update the Apple Pay domain registration flow to use the new Stripe API endpoint.
 * Fix - Fix empty error message for Express Payments when order creation fails.
 * Fix - Fix multiple issues related to the reuse of Cash App Pay tokens (as a saved payment method) when subscribing.
+* Fix - Add missing statement descriptor in Stripe for subscription renewal orders.
```

Comment:
I'm not sure if we want to clarify that its the order number that was missing. This changelog makes it sound as if no statement descriptor was included.

Reply:
Fixed in [c1724ab](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/3451/commits/c1724ab338cae9592fb1c4bb557a546150b4978a) and [2f82887](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/3451/commits/2f82887a339ddf9b3f98e875f380bed889f0041d)